siliconcompiler/apps/sc_dashboard.py

```diff
@@ -40,19 +41,26 @@ def main():
                        'metavar': '<[manifest name, manifest path>'}
     }
 
-    switches = chip.create_cmdline(
-        progname,
-        switchlist=['-loglevel',
-                    '-cfg'],
-        description=description,
-        additional_args=dashboard_arguments)
+    try:
+        switches = chip.create_cmdline(
+            progname,
+            switchlist=[*manifest_switches(),
+                        '-loglevel'],
+            description=description,
+            additional_args=dashboard_arguments)
+    except Exception as e:
+        chip.logger.error(e)
+        return 1
 
     # Error checking
     design = chip.get('design')
     if design == UNSET_DESIGN:
         chip.logger.error('Design not loaded')
         return 1
 
+    if not load_manifest(chip, None):
```

Comment:
Maybe I'm missing something but weren't we loading the manifest before to show the information?
Just wondering because we might be loading the manifest twice now...

Reply:
I'm not sure I follow

Reply:
here we will only load the manifest if it was not provided via the -cfg flag

Reply:
What I meant is: Doesn't `chip.create_cmdline()` already load the manifest from the `cfg`?

Reply:
Sorry misread your answer